db/migrate/20221122162438_create_reminder_emails.rb

```diff
@@ -0,0 +1,8 @@
+class CreateReminderEmails < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reminder_emails do |t|
+      t.references :further_information_request
```

Comment:
```suggestion
      t.references :further_information_request, null: false, foreign_key: true
```

I'd suggest this to ensure data consistency (I'm not really sure why it's not the default 😆).